scale/scheduler/scheduling/scheduling_node.py

```diff
@@ -215,6 +211,7 @@ def reset_new_job_exes(self):
 
         resources = NodeResources()
         for new_job_exe in self._allocated_queued_job_exes:
+            logger.info("IM DEALLOCATING RESOURCES!!!")
```

Comment:
DO WE NEED TO BE SO LOUD? This should probably be a debug statement, as it'll be hit a lot.

Reply:
haha, forgot to clean that one up